source/overview/mattermost-cloud-overview.rst

```diff
@@ -0,0 +1,90 @@
+=========================
+Mattermost Cloud Overview
+=========================
+
+Mattermost Cloud is offered as a monthly subscription service. It is available free, indefinitely, for all workspaces that have 10 or fewer active users. Your subscription is upgraded to pay-as-you-go once you reach 11 or more active users.
+
+How Billing Works
+-----------------
+
+You are billed for active users in your workspace.  An active user: 
+
+- Has created an account and joined a workspace and shows as **Active** in **System Console > Users**.
+- Counts towards your monthly invoice for each day they are active in the billing period.
+
+Users deactivated by a System Administrator are no longer counted toward future monthly invoices.
```

Comment:
@adamjclarkson as a user, I am reading this that the customer will not get billed on the next invoice if they deactivate the user prior to the bill being generated It's a small nuance because I think you do a good job explaining how you are billed. A suggestion to change this might be "Users deactivated by a System Administrator are no longer counted `as active on your next monthly invoice unless you reactive the users within the billing period`"

Reply:
Updated docs to reflect this.